### ISSUE TITLE
When I mount crypt folder with encfsmp in windows, there will be some problem in my kali linux.

### ISSUE THREAD

**Issue by moebiusever** [rhiestan/EncFSMP#2]
Thanks for the works.
I have 2 OS in my laptop. I have 3 disks in my laptop, 1 ntfs for windows , 1 ntfs for data, 1 for kali linux. These is the enviroment:
Windows:
(1)Windows 10 64bit.
(2)Encfsmp 0.10.7
Linux:
(1)Kali Linux Kali GNU/Linux Rolling 64-bit
(2)CyptKeeper 0.9.5
Problem:
(1) Enter the windows.
(2) Open the Encfsmp, and mount the crypt folder which in the data disk.
(3) If I just shutdown the windows without umount the crypt folder, then enter the kali linux, the whole ntfs data disk become readonly.
(4)If I umount the crypt folder and quit the encfsmp before I shutdown the windows, then enter the kali linux, the ntfs data disk can be read and write.

It's strange! Is this a bug?

**Comment by rhiestan** [comment]
Yes, this is strange. Does Linux say why it is mounting the NTFS partition read-only?
It looks like if you shut down Windows without unmounting EncFSMP first, the NTFS disk will be in a bad state.
I fixed a bug in 0.10.7 concerning unmounting while shutting down. Maybe there is still something not done correctly.

**Comment by moebiusever** [comment]
When I shutdown the windows without umount the crypt folder, and then enter the kali linux. Right click the folder in the ntfs with nautils, and see the permissions in the property, it says can be read and write, but in fact, I can't cut or delete any one of the files, the cut and delete menu is gray, if I do this in the terminal with rm -rf, it gives me a readonly filesystem tips. If umount the crypt folder before shutdown the windows, and then enter the kali linux, the cut or delete menu in the ntfs disk becomes normal, it can be used to cut or delete files.
